.python-version

```diff
@@ -0,0 +1,5 @@
+3.9.4
+3.8.9
+3.7.10
+3.6.13
+2.7.18
```

Comment:
For use with [`pyenv`](https://github.com/pyenv/pyenv) to allow easy switching between Python versions